### DISCUSSION TITLE
Episode 141: 🇨🇦 CppNorth Live 🇨🇦 Kate Gregory, Jessica Kerr & Kristen Shaker!

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#32] [Episode]
Episode Link: https://adspthepodcast.com/2023/08/04/Episode-141.html

Feel free to write comments or ask questions 🙂

**Comment by rmerriam** [comment]
Great comments about the conference. It is an amicable and welcoming conference. Glad to speak and attend.
I'm looking forward to next year. 

Kate mentions you learn a lot doing a talk. I know I did. 

Ben commented on talks referencing others. I chuckled when Vincent Ciura showed the book "Functional Programming in C++" by Ivan Cukic because I was also going to mention it in my talk, which was up next. 

"Some human, Matt Godbolt" - hilarious.
